src/gui/rss/feedlistwidget.cpp

```diff
@@ -40,6 +40,28 @@
 #include "base/rss/rss_session.h"
 #include "gui/uithememanager.h"
 
+namespace
+{
+    QIcon tryLoadIcon(const QString &path, const QString &fallbackId)
```

Comment:
`tryDoSomething` looks like a function that indicates an unsuccessful result in some way (for example, it returns a null/invalid value).
So I would drop `try` prefix.